release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -262,6 +262,11 @@ In the following tables, features are marked with the following statuses:
 |Deprecated
 |Deprecated
 
+|Alibaba Cloud clusters
+|Technology Preview
+|Technology Preview
```

Comment:
Shall we insert a line, before the above line 190 (i.e. "Remove"), with the value "Technology Preview"? Thanks!

Reply:
> Shall we insert a line, before the above line 190 (i.e. "Remove"), with the value "Technology Preview"? Thanks!

Wow. Nice catch. :smile: Added Technology Preview to the list.